### DISCUSSION TITLE
Change Operation Handlers are not being triggered

### DISCUSSION THREAD

**Discussion by csamarchi** [eclipse-glsp/glsp#734] [General]
Hello,

I'm working with a Node server and VS Code extension and implementing work based off the [task-list](https://github.com/eclipse-glsp/glsp-examples/tree/00185505c83bef731c5a9435d01ced3d896d95f2/project-templates/node-json-vscode/glsp-server/src) and [workflow](https://github.com/eclipse-glsp/glsp-server-node/tree/main/examples/workflow-server/src) examples.

Hoping you could help me figure out why my handler operations aren't triggering. I'm correctly assigning ID's to my elements. The client is sending, what looks like, correct actions back to the server. The client/server connect appears good. However, nothing changes. 

After an attempt to reposition an element from the server side: 
```
1:04:33 PM info -  [WorkflowGLSPServer] process [action=computedBounds, clientId=workflow-diagram_0] 
1:04:33 PM info -  [WorkflowGLSPServer] process [action=requestPopupModel, clientId=workflow-diagram_0] 
``` 

From client side:
<img width="512" alt="Screen Shot 2022-08-24 at 1 06 07 PM" src="https://user-images.githubusercontent.com/42501494/186491319-9e4626cd-48d6-486d-9607-d8295b59c9dc.png">

Most of the time, it doesn't send any errors. On the occasion, I will get the following:
` [WorkflowGLSPServer] Could not process action: 'requestPopupModel Could not process 'RequestPopupModelAction'. Hover element with id fb2f8726-f51b-49e7-a10b-9a5678ab17a3 not found`

Where exactly does the update of the source model occur? In the handler? 

I'm also curious where in your [workflow example](https://github.com/eclipse-glsp/glsp-server-node/tree/main/examples/workflow-server/src) do you set positions? I don't see a position variable declared in the source model.

Any help is very much appreciated. Thank you

**Comment by tortmayr** [comment]
Hi @csamarchi ,
sorry for the late reply. Somehow, we missed your question. Do you still need help with this?
